appverifier/src/main/java/com/android/mdl/appreader/transfer/TransferManager.kt

```diff
@@ -375,14 +398,12 @@ class TransferManager private constructor(private val context: Context) {
         return verification?.requestToResponseDurationMillis ?: 0
     }
 
-    fun getEngagementMethod(): String {
+    fun getEngagementMethod(): String =
```

Comment:
references all `ENGAGEMENT_METHOD_` int types in `EngagementTypeDef` (that previously used to live in this file)